src/Model/SectionData/LoginSectionDataService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Hyva\Checkout\Model\SectionData;
```

Comment:
The namespace is unnecessarily nested and (for me) confusing. Given that we don't have anything else in `Model` yet and `SectionData` is similar to `CustomerData` where the actual section data providers are, I'd suggest a simple `Hyva\Checkout\Model\LoginSectionDataService`

Reply:
Okay, alright. I will make changes accordingly.